### ISSUE TITLE
Why does position_ids start at 0-index?

### ISSUE THREAD

**Issue by Viredery** [Physical-Intelligence/openpi#762]
I saw the code: `positions = jnp.cumsum(input_mask, axis=1) - 1` in pi0.py

But in PaliGemma, the start index is 1:

https://github.com/huggingface/transformers/blob/main/src/transformers/models/paligemma/modeling_paligemma.py#L391

I'm curious the reason of the difference.
